app/pages/resource/info/ResourceInfo.spec.js

```diff
@@ -20,13 +32,53 @@ describe('pages/resource/info/ResourceInfo', () => {
     return shallow(<ResourceInfo {...defaults} {...props} />);
   }
 
-  it('renders unit name', () => {
-    const unitName = getWrapper().find('.unit-name');
-    expect(unitName.text()).to.equal(unit.name.fi);
+  describe('header', () => {
+    let header;
+    before(() => {
+      header = getWrapper().find('header');
+    });
+
+    it('renders unit name', () => {
+      const unitName = header.find('.unit-name');
+      expect(unitName.text()).to.equal(unit.name.fi);
+    });
+
+    it('renders resource name', () => {
+      const resourceName = header.find('.resource-name');
+      expect(resourceName.text()).to.equal(resource.name.fi);
+    });
+
+    it('renders resource address', () => {
+      const unitAddress = header.find('.unit-address');
+      expect(unitAddress.text()).to.contain(unit.streetAddress.fi);
+    });
   });
+  describe('resource-details section', () => {
+    let section;
+    before(() => {
```

Comment:
- [ ] Oskari had a good point in my PR that instead of using before for these kind of things we should define new get...Wrapper functions if it's simple to do. So in this case it would be
```javascript
function getSectionWrapper() {
  return getWrapper().find('section.resource-details');
}
```
And the tests would use it like this:
```javascript
it('renders a ImageCarousel with resource images', () => {
  const imageCarousel = getSection().find(ImageCarousel);
  ...
```
With these kind of wrappers it's also easy to add test cases for different params in more complicated cases.
The wrapper can be defined in the describe block where it is needed.